plugins/database/src/db_plugin.cpp

```diff
@@ -6520,12 +6572,14 @@ irods::error db_check_auth_op(
     getNowStr( myTime );
 
     /* Check for PAM_AUTH type passwords */
-    pamMaxTime = atoll( irods_pam_password_max_time );
-    pamMinTime = atoll( irods_pam_password_min_time );
 
-    if ( ( strncmp( goodPwExpiry, "9999", 4 ) != 0 ) &&
-            expireTime >=  pamMinTime &&
-            expireTime <= pamMaxTime ) {
+    if (const auto err = get_auth_config("authentication::pam_password", ac); !err.ok()) {
```

Comment:
It will not. I think we need to decide on a direction here, because it feels like we might start moving in a different direction: https://github.com/irods/irods/issues/7318 I realize that issue has to do with config *files* but it is a configuration nonetheless.